pkg/flowkit/state_test.go

```diff
@@ -151,7 +151,99 @@ func generateComplexProject() State {
 
 	return *p
 }
+func generateComplexProjectSameAddress() State {
```

Comment:
You don't need to duplicate this necessarily, you can just put another account with same address and different name under `generateComplexProject`.

Reply:
addressed